### PR TITLE
Add missing elements of SQA document templates

### LINKED ISSUE #127: Add Software Quality Assurance documents
## Reason
Software quality assurance practices and documentation assure users that care is taken in the development of a code.

## Design
Utilize the templates from MOOSE to create the initial sqa documents in Blackbear as a first step. Evaluate and add BlackBear specific information as a second step, if necessary

## Impact
Bring BlackBear into compliance with the SQA practices used in the MOOSE-herd

### PR DESCRIPTION
**Pull request by bwspenc**
closes #127

### CODE REVIEW

**Comment by cticenhour** [comment]
Thanks @bwspenc! I will put this on my list to look at tomorrow.

File: doc/content/sqa/blackbear_srs.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+!! system-purpose-begin
+The purpose of BlackBear is to simulate the progression of degradation in structural components (e.g., buildings, bridges, dams) due to exposure to the environmental conditions they experience over their lifetime. In addition, BlackBear predicts the response of those structures in the presence of that long-term degradation to loading conditions of interest to assess the effects of degradation on their integrity.
+
+!! system-purpose-finish
+!template-end!
+
+!template! item key=assumptions-and-dependencies
+{{app}} has no constraints on hardware and software beyond those of the MOOSE framework and modules listed in their respective SRS documents, which are accessible through the links at the beginning of this document.
+
+{{app}} provides access to a number of code objects that perform computations such as material behavior and boundary conditions. These objects each of makes their own physics-based assumptions, such as the units of the inputs and outputs. Those assumptions are described in the documentation for those individual objects.
```
Comment (cticenhour) [inline doc/content/sqa/blackbear_srs.md:23]:
```suggestion
{{app}} provides access to a number of code objects that perform computations such as material behavior and boundary conditions. These objects each make their own physics-based assumptions, such as the units of the inputs and outputs. Those assumptions are described in the documentation for those individual objects.
```

File: doc/content/sqa/blackbear_srs.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+!template-end!
+
+!template! item key=user-characteristics
+{{app}} has three main classes of users:
+
+- +{{app}} Developers+: These are the core developers of {{app}} module. They are responsible for designing, implementing, and maintaining the software, while following and enforcing its software development standards.
```
Comment (cticenhour) [inline doc/content/sqa/blackbear_srs.md:29]:
```suggestion
- +{{app}} Developers+: These are the core developers of {{app}}. They are responsible for designing, implementing, and maintaining the software, while following and enforcing its software development standards.
```

File: doc/content/sqa/blackbear_srs.md [hunk 3]
```diff
@@ -1,5 +1,36 @@
...
+- +Developers+: These are scientists or engineers that modify or add capabilities to {{app}} for their own purposes, which may include research or extending its capabilities. They will typically have a background in structural or mechanical engineering, and in modeling and simulation techniques, but may have more limited background in code development using the C++ language. In many cases, these developers will be encouraged to contribute code back to {{app}}.
+- +Analysts+: These are users that run {{app}} to run simulations, but do not develop code. The primary interface of these users with {{app}} is the input files that define their simulations. These users may interact with developers of the system requesting new features and reporting bugs found.
+!template-end!
+
```
Comment (cticenhour) [inline doc/content/sqa/blackbear_srs.md:33]:
```suggestion

!template! item key=information-management
{{app}} as well as the core MOOSE framework in its entirety will be made publicly available on an appropriate repository
hosting site. Day-to-day backups and security services will be provided by the hosting service. More
information about backups of the public repository on [!ac](INL)-hosted services can be found
on the following page: [sqa/github_backup.md]
!template-end!

!template! item key=policies-and-regulations
!include framework_srs.md start=policies-and-regulations-begin end=policies-and-regulations-finish
!template-end!

!template! item key=packaging
No special requirements are needed for packaging or shipping any media containing [!ac](MOOSE) and
the {{app}} source code. However, some [!ac](MOOSE)-based applications
that use the {{app}} code may be export-controlled, in which case all export control restrictions must
be adhered to when packaging and shipping media.
!template-end!

```

A couple missing sections - I'll go ahead and fill them in.

**Review by cticenhour — CHANGES_REQUESTED** [review]

**Review by cticenhour — APPROVED** [review]

**Comment by cticenhour** [comment]
@bwspenc This looks great to me! I don't have CCB authority in BlackBear, however, so someone else will need to step in for an official approval. Perhaps @lynnmunday?

**Review by lynnmunday — APPROVED** [review]

**Pull request merged**